modules/appeals_api/spec/support/schema_helpers.rb

```diff
@@ -43,6 +43,6 @@ def override_max_lengths(appeal, schema)
   private
 
   def shared_schema_dir(file)
-    Rails.root.join('modules', 'appeals_api', Settings.modules_appeals_api.schema_dir, 'shared', 'v1', file)
+    Rails.root.join('modules', 'appeals_api', Settings.modules_appeals_api.schema_dir, 'shared', 'v0', file)
```

Comment:
Similar to my note above, we hadn't had any tests yet that used this path, so this incorrect value from when I was moving fixtures around earlier was missed until now